docs/Cluster-VictoriaMetrics.md

```diff
@@ -315,20 +315,24 @@ The metric is set to `0` when the `vmstorage` isn't in read-only mode.
 
 ## URL format
 
+##### Differences in the URL structures for data ingestion between standalone/cluster
+A note on the differences between the cluster version and standalone version URL structures related to data ingestion.  In the standalone version where the import time series data formats for `csv`, `native` and `prometheus` are `/api/vi/import/<format>` (apart from `jsonl` which has no suffix `api/vi/import`). In the cluster version all endpoints to import time series data are prefixed with `/prometheus` so they become `/prometheus/api/vi/import/<format>`, apart from the `jsonl` format which also has no suffix in the cluster version `/prometheus/api/vi/import`)
```

Comment:
This paragraph is related only to the part of API, but precedes the full reference. I don't think it makes things clearer. 
In my opinion, referring to the difference between Single server and cluster versions would be logical:
```
The main differences between URL formats of cluster and [Single server](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html)
versions are that cluster has separate components for read and ingestion path, and because of multi-tenancy support.
Check practical examples of VictoriaMetrics API [here](https://docs.victoriametrics.com/url-examples.html).
```

Using `#` tags affects the navigation - docs engine uses them for building the hierarchical nav bar on the left of the https://docs.victoriametrics.com/. I'd like not to change it and put the paragraph just after the `## URL format` header.

Reply:
@hagen1778 :+1: That makes sense.
I did not know/find that https://docs.victoriametrics.com/url-examples.html page, which does indeed show all the methods.  I still think a user that has used the standalone version and starts using the cluster version is possibly going to be confused and not understand the fact that now `/prometheus/api/v1/import` endpoint does not only handles Prometheus data but multiple data formats - csv, json and native.  That is very confusing.

As a user when you are reading docs and you see `/prometheus` this can often make you blind to the lines or infer that this is not want you are looking for because it is for **prometheus data** and we scan over that and do not read the fine details.  This is human :)

So may I suggest:

```
The main differences between URL formats of cluster and [Single server](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html)
versions are that cluster has separate components for read and ingestion path, and because of multi-tenancy support.
Also in the cluster version the `/prometheus/api/v1` endpoint ingests  `jsonl`, `csv`, `native` and `prometheus` data formats **not** only `prometheus` data.
Check practical examples of VictoriaMetrics API [here](https://docs.victoriametrics.com/url-examples.html).
```
I think the purpose here is to be explicit and advise/hint to the user that the `/prometheus/api/v1/import` endpoint does multiple roles where in standalone it does not serve those roles, that is the part that is not specifically made clear.

Updating PR.

Reply:
@valyala @zekker6 @f41gh7 wdyt?